frontend/src/app/organization/organization-details/organization-details.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+import { Organization } from '../organization.model';
+
+@Component({
+  selector: 'volontulo-organization-details',
+  templateUrl: './organization-details.component.html',
+  styleUrls: ['./organization-details.component.css'],
```

Comment:
This component can most likely utilize OnPush detection strategy. See here: https://angular-2-training-book.rangle.io/handout/change-detection/change_detection_strategy_onpush.html